src/store/fileUploadStore.js

```diff
@@ -211,6 +211,8 @@ const actions = {
 			let localUrl = ''
 			if (file.type === 'image/png' || file.type === 'image/gif' || file.type === 'image/jpeg') {
 				localUrl = URL.createObjectURL(file)
+			} else if (isVoiceMessage) {
```

Comment:
add this arg to the JSDoc and extend the unit test fileUploadStore.spec.js ?

Reply:
@PVince81 I'll create an issue for the tests and I'll look into it once I have finished all the urgent tasks